server/service/src/sync/translations/preference.rs

```diff
@@ -42,11 +42,10 @@ impl SyncTranslation for PreferenceTranslator {
         row: &ChangelogRow,
         r#type: &ToSyncRecordTranslationType,
     ) -> bool {
-        match r#type {
-            ToSyncRecordTranslationType::PullFromOmSupplyCentral => {
-                self.change_log_type().as_ref() == Some(&row.table_name)
-            }
-            _ => false,
+        if let ToSyncRecordTranslationType::PullFromOmSupplyCentral = r#type {
```

Comment:
I think this would just be `if` rather than `if let`

Though maybe best to leave this as match statement, I imagine pushing at least some variants of prefs from remote sites back to oms central will happen at some point 👀

Reply:
I agree, I think these refactors should be avoided in general (reduce diff reduce stylistic discussions), unless we've decided to only go with one pattern (but even then, patch update probably not the best time to apply those new patterns)

Reply:
reverted